Testscripts/Windows/NTTTCP-MONITOR-THROUGHPUT.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think we have the ground rule for PS script in LISAv2. One of them is using try-catch blocks. We put the exception print statement. Can you apply to your script?

Reply:
The framework has already wrapped with a well try.. catch, for tests, don't need to write duplicate ones. just focus logic.

Reply:
That pattern helps to identify and print out the exception in the test case level. If the test runtime raises the exception, is it escalated to the framework's catch block and print out the exception message?